Userland/Libraries/LibJS/Heap/Cell.h

```diff
@@ -73,6 +74,63 @@ class Cell {
             visit_impl(const_cast<RemoveConst<T>&>(*cell.ptr()));
         }
 
+        template<typename T>
+        void visit(ReadonlySpan<T> span)
+        {
+            for (auto& value : span)
+                visit(value);
+        }
+
+        template<typename T>
+        void visit(Span<T> span)
+        {
+            for (auto& value : span)
+                visit(value);
+        }
+
+        template<typename T>
+        void visit(Vector<T> const& vector)
+        {
+            for (auto& value : vector)
+                visit(value);
+        }
```

Comment:
I'd suspect we don't need this one? The span overloads should cover Vector, like they're doing for e.g. Array

Reply:
Alas it seems we do indeed need it